src/Network/Network/Cortex/VpnSite/NewAzVpnSiteLinkCommand.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.Network.Models;
+    using System;
+
+    [Cmdlet(
+        VerbsCommon.New,
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VpnSiteLink",
+        SupportsShouldProcess = false),
+        OutputType(typeof(PSVpnSiteLink))]
+    public class NewAzVpnSiteLinkCommand : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "Name")]
```

Comment:
Please clarify, which resource's name it is

Reply:
done